Source/App.js

```diff
@@ -1 +1,340 @@
+
+        // Get the array of entities
+        var neighborhoodEntities = dataSource.entities.values;
+        for (var i = 0; i < neighborhoodEntities.length; i++) {
+            var entity = neighborhoodEntities[i];
+
+            if (Cesium.defined(entity.polygon)) {
+                // Use kml neighborhood value as entity name
+                entity.name = entity.properties.neighborhood;
+                // Set the polygon material to a random, translucent color
+                entity.polygon.material = Cesium.Color.fromRandom({
+                    red : 0.1,
+                    maximumGreen : 0.5,
+                    minimumBlue : 0.5,
+                    alpha : 0.6
+                });
+                // Generate Polygon center
+                var polyPositions = entity.polygon.hierarchy.getValue(Cesium.JulianDate.now()).positions;
+                var polyCenter = Cesium.BoundingSphere.fromPoints(polyPositions).center;
+                polyCenter = Cesium.Ellipsoid.WGS84.scaleToGeodeticSurface(polyCenter);
+                entity.position = polyCenter;
+                // Generate labels
+                entity.label = {
+                    text : entity.name,
+                    showBackground : true,
+                    scale : 0.6,
+                    horizontalOrigin : Cesium.HorizontalOrigin.CENTER,
+                    verticalOrigin : Cesium.VerticalOrigin.BOTTOM,
+                    distanceDisplayCondition : new Cesium.DistanceDisplayCondition(10.0, 8000.0),
+                    disableDepthTestDistance : Number.POSITIVE_INFINITY
+                };
+            }
+        }
+    });
+
+    // Load a drone flight path from a CZML file
+    var dronePromise = Cesium.CzmlDataSource.load('./Source/SampleData/SampleFlight.czml');
+
+    // Save a new drone model entity
+    var drone;
+    dronePromise.then(function(dataSource) {
+        viewer.dataSources.add(dataSource);
+        drone = dataSource.entities.values[0];
+        // Attach a 3D model
+        drone.model = {
+            uri : './Source/SampleData/Models/CesiumDrone.gltf',
+            minimumPixelSize : 128,
+            maximumScale : 2000
+        };
+        // Add computed orientation based on sampled positions
+        drone.orientation = new Cesium.VelocityOrientationProperty(drone.position);
+
+        // Smooth path interpolation
+        drone.position.setInterpolationOptions({
+            interpolationAlgorithm : Cesium.HermitePolynomialApproximation,
+            interpolationDegree : 2
+        });
+        drone.viewFrom = new Cesium.Cartesian3(30, 0, 0);
+    });
+
+    //////////////////////////////////////////////////////////////////////////
+    // Load 3D Tileset
+    //////////////////////////////////////////////////////////////////////////
+
+    // Load the NYC buildings tileset
+    var city = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+        url: 'https://cesiumjs.org/NewYork/3DTilesGml',
```

Comment:
Use the composer link instead: 'https://beta.cesium.com/api/assets/1461?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiJkYWJmM2MzNS02OWM5LTQ3OWItYjEyYS0xZmNlODM5ZDNkMTYiLCJpZCI6NDQsImFzc2V0cyI6WzE0NjFdLCJpYXQiOjE0OTkyNjQ3NDN9.vuR75SqPDKcggvUrG_vpx0Av02jdiAxnnB1fNf-9f7s'